### PR TITLE
Fix #10 #11 #12

### LINKED ISSUE #10: No images for questions
Not sure if this is intentional, but there are no images in the questions section.
image

### PR DESCRIPTION
**Pull request by hnasheralneam**
Moves all content over from the old site. The next time I get some down time, I'll make the teams page. Also, I'm very very sorry for the inline styles, but I've got no idea how svelte works and thus could not find the stylesheets.

### CODE REVIEW

**Comment by iamawatermelo** [comment]
Styles are at the bottom of the page. I'll just fix it myself and merge.

File: src/routes/code-of-conduct/+page.svelte
Comment (iamawatermelo) [inline src/routes/code-of-conduct/+page.svelte:1]: This file is already in another PR

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -98,36 +98,35 @@
...
   <div>
     <div>
+  <h2>What is Zenith?</h2>
       <p>
         Zenith 2025 is a hackathon in San Francisco organised by and for teens. Join us for 2 days to share and
         collaborate with like minded individuals in a supportive, mistake friendly environment.
       </p>
     </div>
-    <div class="image-box"></div>
+    <div><img class="image-box" src="https://cdn.britannica.com/41/196341-050-6B55B7E7/solar-eclipse-moon-sun-space-astronomy.jpg" alt="A photo of a solar eclipse"></div>
```
Comment (iamawatermelo) [inline src/routes/+page.svelte]: Do not hotlink to images + accessibility: screen readers already announce that it is an image

File: src/routes/+layout.svelte [hunk 2]
```diff
@@ -43,26 +43,49 @@
...
       </div>
+      
+    </div>
+    <div style="margin: 2rem 10rem 0 10rem">
+            <p>
+          Hack Zenith 2025 is fiscally sponsored by The Hack Foundation (d.b.a. Hack Club), a 501(c)(3) nonprofit (EIN:
+          81-2908499). Hack Zenith 2025 is not an official Hack Club event. Use of Hack Club branding does not
+          constitute an endorsement, implied or otherwise, of Hack Zenith 2025 by Hack Club.
+        </p>
+
```
Comment (iamawatermelo) [inline src/routes/+layout.svelte:87]: Why was this moved?

File: src/routes/+layout.svelte [hunk 3]
```diff
@@ -43,26 +43,49 @@
...
+        <p class="title">Social</p>
         <ul>
-          <li><a href="https://hackclub.com/slack/">Join us on the Hack Club Slack</a></li>
-          <li><a href="https://github.com/zenith-hacks/zenith">View this page's source code</a></li>
-          <li><a href="https://hcb.hackclub.com/zenithhacks">Look at our financials</a></li>
+         <p>
+            <span  style="color: #999">Instagram</span> <a href='https://instagram.com/zenith_hacks'>@zenith_hacks</a>
+         </p>
+         <p>
+            <span style="color: #999">Threads</span> <a href='https://threads.net/@zenith_hacks'>@zenith_hacks</a>
```
Comment (iamawatermelo) [inline src/routes/+layout.svelte]: Please use the same style as everything else

File: src/routes/+page.svelte [hunk 4]
```diff
@@ -98,36 +98,35 @@
...
 <article class="info info-right">
-  <h2>What's a hackathon?</h2>
   <div>
     <div>
+  <h2>What is a hackathon?</h2>
       <p>
         <!-- TODO: Change this -->
-        A hackathon is a social event focused on coding where teenagers come together to learn, build, and share their results
-        with others!
+        A hackathon is a social event focused on coding where teenagers come together to learn, build, and share their results with others!
```
Comment (iamawatermelo) [inline src/routes/+page.svelte:119]: Please format

File: src/lib/logos/desmos-on-light.png
Comment (iamawatermelo) [inline src/lib/logos/desmos-on-light.png:1]: We aren't sponsored by the Desmos Foundation.

File: src/lib/logos/1password-on-light.svg
Comment (iamawatermelo) [inline src/lib/logos/1password-on-light.svg:1]: This file already exists.

File: src/lib/logos/codeforcause.png
Comment (iamawatermelo) [inline src/lib/logos/codeforcause.png:1]: This file already exists.

File: src/lib/logos/disney-on-light.svg
Comment (iamawatermelo) [inline src/lib/logos/disney-on-light.svg:1]: This file already exists.

File: src/lib/logos/google.png
Comment (iamawatermelo) [inline src/lib/logos/google.png:1]: We aren't sponsored by Google.

File: src/lib/logos/interview-buddy.png
Comment (iamawatermelo) [inline src/lib/logos/interview-buddy.png:1]: Where is this on the sponsorship spreadsheet?

File: src/lib/logos/redbull.svg
Comment (iamawatermelo) [inline src/lib/logos/redbull.svg:1]: This file already exists.

**Comment by hnasheralneam** [comment]
Thanks for fixing it! I'll make sure to put them in the right place next time.

**Review by iamawatermelo — CHANGES_REQUESTED** [review]
This PR contains many duplicated images and changes too many things, including styling which should be coordinated with me first. Please read the README. Try find SVG logos for our sponsors---I've asked everyone to get in contact with their sponsor contacts to get this.

File: src/routes/code-of-conduct/+page.svelte
Comment (hnasheralneam) [reply src/routes/code-of-conduct/+page.svelte:1]: Did not see it, sorry.

File: src/routes/+page.svelte (hunk 1 above)
Comment (hnasheralneam) [reply src/routes/+page.svelte]: I'll add the image to the project, alt text is still important to know what the image is about.

File: src/routes/+layout.svelte (hunk 2 above)
Comment (hnasheralneam) [reply src/routes/+layout.svelte:87]: I was attempting to put the footer links into clearer categories.

File: src/routes/+layout.svelte (hunk 3 above)
Comment (hnasheralneam) [reply src/routes/+layout.svelte]: All other footer links are using the same style, it makes it clear what part is the link, and it's easier to skim than a sentence description for each.

File: src/routes/+page.svelte (hunk 4 above)
Comment (hnasheralneam) [reply src/routes/+page.svelte:119]: Sorry, I'll try to configure the IDE to your style (2 chars spaces?) and format it.

File: src/lib/logos/desmos-on-light.png
Comment (hnasheralneam) [reply src/lib/logos/desmos-on-light.png:1]: I just copied all the images over, I forgot to remove it. I only used the Desmos company logo in the sponsors section.

File: src/lib/logos/1password-on-light.svg
Comment (hnasheralneam) [reply src/lib/logos/1password-on-light.svg:1]: Sorry, I'll remove all the extra image files.

File: src/lib/logos/google.png
Comment (hnasheralneam) [reply src/lib/logos/google.png:1]: They offered us the location, and for now it seems like the main choice. We can change it if we settle on somewhere else.

File: src/lib/logos/interview-buddy.png
Comment (hnasheralneam) [reply src/lib/logos/interview-buddy.png:1]: Rushil told me to add it through the slack, I don't know if he added it to the sheet.

**Comment by hnasheralneam** [comment]
We can't just not include sponsors because there is no SVG version of their logo currently available; the PNG logos work for now, and the .2 seconds they add to the page load seem like a fair price to pay for the support the companies are giving us.

**Comment by iamawatermelo** [comment]
Alright. Please resolve all changes though, you've just marked them as resolved without doing anything...

**Comment by hnasheralneam** [comment]
I've made (some of) the changes you asked for.

**Comment by hnasheralneam** [comment]
@iamawatermelo I've resolved the conflicts, can you look at this again?

**Pull request closed without merging**